waiter/specs/k8s-default-pod.edn

```diff
@@ -0,0 +1,41 @@
+{:kind "ReplicaSet"
```

Comment:
I think it would be nice having the ability to patch things in the pod template spec (e.g., add a new environment variable, or update the image uri) without having to rebuild the JAR file. Is there a reason why we wouldn't want to treat this like another configuration file?

Reply:
The path to the pod spec file is now part of the `:kubernetes` settings in our Waiter config file.